### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hanahmily**
###  What have you changed?

There's an error document link in README.md. The PR intends to sync it with latest docs website.

###  What is the type of the changes?

Pick one of the following and delete the others:

- Bugfix (a change which fixes an issue)

###  How is the PR tested?
Please select the tests that you ran to verify your changes:

- No code

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ TiKV has the following key features:
 
 ## Documentation
 
-For instructions on deployment, configuration, and maintenance of TiKV,see TiKV documentation on our [website](https://tikv.org/docs/3.0/tasks/getting-started/). For more details on concepts and designs behind TiKV, see [Deep Dive TiKV](https://tikv.org/docs/deep-dive/introduction/).
+For instructions on deployment, configuration, and maintenance of TiKV,see TiKV documentation on our [website](https://tikv.org/docs/3.0/tasks/try/). For more details on concepts and designs behind TiKV, see [Deep Dive TiKV](https://tikv.org/docs/deep-dive/introduction/).
```
Comment (zhangjinpeng1987) [inline README.md]: Maybe https://tikv.org/docs/3.0/tasks/introduction/ is better?

Comment (hanahmily) [reply README.md]: make sense. done.

File: README.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ TiKV has the following key features:
 
 ## Documentation
 
-For instructions on deployment, configuration, and maintenance of TiKV,see TiKV documentation on our [website](https://tikv.org/docs/3.0/tasks/getting-started/). For more details on concepts and designs behind TiKV, see [Deep Dive TiKV](https://tikv.org/docs/deep-dive/introduction/).
+For instructions on deployment, configuration, and maintenance of TiKV,see TiKV documentation on our [website](https://tikv.org/docs/3.0/tasks/introduction/). For more details on concepts and designs behind TiKV, see [Deep Dive TiKV](https://tikv.org/docs/deep-dive/introduction/).
```
Comment (siddontang) [inline README.md]: PTAL @dcalvin

**Pull request merged**